web: add support for colornote importer (#9273)
streetwriters/notesnook: docs/help/contents/importing-notes/import-notes-from-colornote.md

---
title: ColorNote
---

> 

Reviewer: I noticed there is a TODO in importer.ts mentioning that trashed notes and reminders are not supported yet. It might be good to reflect this limitation in the documentation as well, so users are aware of it before importing.

Author: the TODOs have been completed. We don't import trashed notes so support for that isn't added. Support for reminders added.

# How to import notes from ColorNote notes app?

The following steps will help you import your notes from ColorNote easily.

1. Download the ColorNote mobile app.
2. Go to `Settings > Backup` and create a backup. Make sure to remember the password you set while creating the backup as it will be needed during import.
3. Open the Notesnook app (web or desktop).
4. Go to `Settings > Notesnook Importer` and select `ColorNote` from list of apps.
5. Drop the backup file you exported earlier in the box or click anywhere to open system file picker to select the backup. 

> 7. In the "colornote password" text input, enter the password you used while creating the backup. Then, click on "Start importing".

Reviewer 2: ```suggestion
7. Click on "Start importing" and enter the password you used while creating the backup in the popup.
```

> 7. Once importing completes you should see all your notes in Notesnook. If you face any issues during importing, [report it on github](https://github.com/streetwriters/notesnook).

Reviewer: ```suggestion
7. Once importing completes you should see all your notes in Notesnook. If you face any issues during importing, [report it on github](https://github.com/streetwriters/notesnook). (Note: reminders and trashed notes are not imported yet. Support for these features is planned.)
```
